voxblox/src/integrator/tsdf_integrator.cc

```diff
@@ -66,67 +67,51 @@ inline TsdfVoxel* TsdfIntegratorBase::findOrTempAllocateVoxelPtr(
   if (block_idx != *last_block_idx) {
     *last_block = layer_->getBlockPtrByIndex(block_idx);
     *last_block_idx = block_idx;
-    if (*last_block != nullptr) {
-      (*last_block)->updated() = true;
-    }
   }
 
   // If no block at this location currently exists, we allocate a temporary
   // voxel that will be merged into the map later
```

Comment:
you now allocate a temporary block, right?

Reply:
Yes, because of this setting the updated flag has been moved out of the if, as the block will now always exist